ccm_web/server/src/api/api.enroll.section.users.handler.ts

```diff
@@ -49,14 +49,18 @@ export class EnrollSectionUsersApiHandler {
   }
 
   async enrollUser (user: SectionUserDto): Promise<CanvasEnrollment | APIErrorData> {
+    const enrollLoginId = user.loginId
+      .replace(/@umich\.edu$/i, '')
```

Comment:
Does there need to be something to handle @med.umich.edu (and if there are similar prefixes like this?)
I don't know the rules for the med accounts, just aware that they exist.

Reply:
I really don't know.

Maybe @pushyamig or @melindakraft know.

Reply:
I suppose in the way we handle `@umich.edu` addresses, by dropping the domain, we could do the same thing for `@med.umich.edu`.  Because I think med. campus people are usually identified by their uniqname.

Reply:
Admittedly, I had suggested along the way (I think in the description of #108) that the domain to remove, like `@umich.edu` should be configurable.  But then I ignored that when I decided to make it a regex that's anchored to the end of the string and case is ignored.  I thought that maybe we don't want to throw regexes into the app. config.  (Maybe I'm wrong.)

Alternatively, I could reimplement that part to not use regex.  That's probably a little "safer".  It may also make it easier to support a _list_ of domains to be stripped from the login ID. 🤷‍♂️

Reply:
@chrisrrowland, thanks for reviewing again and raising this excellent question.  I'm sorry I didn't notice your new review before I requested a new one from you!

I've been busy with crunch time support, so I didn't realize until now that you're probably back with IQ full time, right?

Reply:
> I suppose in the way we handle `@umich.edu` addresses, by dropping the domain, we could do the same thing for `@med.umich.edu`. Because I think med. campus people are usually identified by their uniqname.

I was curious if the uniqname part was distinct across all *umich.edu email addresses.
If so it seems reasonable to strip anything that ends .umich.edu down to just the uniqname.
I would guess it works this way and nobody can come in and create rowc@med.umich.edu

If not, i suppose it would just be treated as a normal email address, b/c how else would you distinguish...

I'm actually still on loan until the end of the month.. not rid of me yet

Reply:
AFAIK, a user cannot have `name@med.umich.edu` without also having `name@umich.edu`. The `med` domain will not have anything that `umich.edu` will not.  We're also opening a can of worms here, I think, because we may have to start considering other domains like `engin.umich.edu` and others.

Reply:
I've not added code to handle the `med.umich.edu` addresses yet.  That may be an overoptimization.  We don't really know whether our users will need it.  (We don't even know whether they will need the `umich.edu` handling, either.)

I will open a new issue to consider handling a list of email domains to strip and to make it configurable.

Reply:
Opened #192.